aten/src/ATen/core/function_schema.h

```diff
@@ -49,7 +49,6 @@ struct Argument {
         is_inferred_type_(is_inferred_type) {
     if (default_value_ && default_value_->isTensor()) {
       auto t = default_value_->toTensor();
```

Comment:
@ezyang what's the point of keeping the if statement if the assert does not exists? Is there some side effect that is not commented here?

Reply:
You're right, there's no point.